src/modules/dashboard/components/CoinMachine/DeleteCommentDialog/DeleteCommentDialog.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+
+import { FormattedMessage, defineMessages } from 'react-intl';
+
+import Button from '~core/Button';
+import Dialog, { DialogProps, DialogSection } from '~core/Dialog';
+import Heading from '~core/Heading';
+import Comment, { Props as CommentProps } from '~core/Comment';
+
+import styles from './DeleteCommentDialog.css';
+
+const MSG = defineMessages({
+  title: {
+    id: 'dashboard.CoinMachine.DeleteCommentDialog.title',
+    defaultMessage: 'Delete comment',
+  },
+  question: {
+    id: 'dashboard.CoinMachine.DeleteCommentDialog.question',
+    defaultMessage: `Are you sure you want to delete this message?`,
+  },
+  buttonCancel: {
+    id: 'dashboard.CoinMachine.DeleteCommentDialog.buttonCancel',
+    defaultMessage: 'Cancel',
+  },
+  buttonDelete: {
+    id: 'dashboard.CoinMachine.DeleteCommentDialog.buttonDelete',
+    defaultMessage: 'Delete',
+  },
+});
+
+const displayName = 'dashboard.CoinMachine.DeleteCommentDialog';
+
+interface Props extends DialogProps {
+  comment: CommentProps;
+}
+
+const DeleteCommentDialog = ({ cancel, comment }: Props) => {
```

Comment:
This does not live inside Coin Machine, as it a general, `core` level component.

For that it needs to be both moved and renamed:
- Move it to `src/modules/core`
- Rename it to `CommentDeleteDialog` _(all our other core level comment components have the `Comment` prefix)_